src/main/java/com/novi/springboottechiteasy/models/CiModule.java

```diff
@@ -0,0 +1,55 @@
+package com.novi.springboottechiteasy.models;
+
+import jakarta.persistence.*;
+
+import java.util.List;
+
+@Entity
+public class CiModule {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String brand;
+    private String type;
+    private Double price;
+
+    @OneToMany(mappedBy = "compatibleModule", cascade = CascadeType.ALL)
```

Comment:
Je gebruikt hier cascade, waardor mijn opmerking in CiModuleService, over het leggen van de relatie, weleens irrelevant zou kunnen zijn.